package.json

```diff
@@ -41,7 +41,6 @@
     "applicationinsights": "^0.19.0",
     "chalk": "^1.1.3",
     "commander": "^2.9.0",
-    "node-status": "^1.0.0",
```

Comment:
It does - see https://www.npmjs.com/package/node-status, where the developer comments, "Right now, if you have to use console.log after the status bar has started, it can be a bit janky because the stdout line isn't cleared when a console.log is run." When I tried the office validator on one machine, it consistently hung after validation unless I removed the node-status calls. Our only use of the node-status module is to start a spinner and then remove it once we're done, without any other updates, which isn't really that helpful, so removing it is probably the best course of action.